internal/dbmodel/modelmigration.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2025 Canonical.
+
+package dbmodel
+
+import (
+	"database/sql"
+	"time"
+)
+
+// ModelMigration holds the information for a pending model migration.
+type ModelMigration struct {
```

Comment:
This needs a different name, this isn't model migrations, this is "ModelMigrationTargetAndInitialUserMappingsToStoreOnCompletion" (Just describing it aha, but a better name is my point) and I also think you should godoc its explicit purpose (specifically UserMapping) and how it'll be used. We'll lose context in time with this, so I'd prefer us be kind to ourselves

Reply:
Here are some name options `ActiveModelMigrations` or `ModelMIgrationJobs` or `OngoingModelMigrations`. But I'm not sure about including details about the fields in the table in the name.

Reply:
I've updated the godoc though.

Reply:
Whatever you think if best, I just didn't think calling it ModelMigration was right. Happy with any of them honestly.